frontend/src/scenes/persons/PersonV2.tsx

```diff
@@ -0,0 +1,165 @@
+                                                <Tag className="extra-ids">
+                                                    +{person.distinct_ids.length} <DownOutlined />
+                                                </Tag>
+                                            </Dropdown>
+                                        )}
+                                    </div>
+                                </div>
+                                <div className="item-group">
+                                    <label>First seen</label>
+                                    <div>{moment(person.created_at).fromNow()}</div>
+                                </div>
+                                <div className="text-center mt">
+                                    <a onClick={() => setMergeModalOpen(true)}>
+                                        <MergeCellsOutlined /> Merge person
+                                    </a>
+                                </div>
+                                <div className="text-center mt">
+                                    <Popconfirm
+                                        title="Are you sure to delete this person and all associated data?"
+                                        onConfirm={deletePerson}
+                                        okText="Yes"
+                                        cancelText="No"
+                                    >
+                                        <Button
+                                            onClick={() => console.log(1)}
+                                            className="text-danger"
+                                            disabled={deletedPersonLoading}
+                                            data-attr="delete-person"
+                                            type="link"
+                                        >
+                                            {deletedPersonLoading ? <LoadingOutlined spin /> : <DeleteOutlined />}{' '}
+                                            Delete this person
+                                        </Button>
+                                    </Popconfirm>
+                                </div>
+                            </>
+                        )}
+                        {!person && personLoading && <Skeleton paragraph={{ rows: 4 }} active />}
+                    </Card>
+                    <Card className="card-elevated person-properties" style={{ marginTop: 16 }}>
+                        <Tabs>
+                            <TabPane
+                                tab={<span data-attr="persons-properties-tab">Properties</span>}
+                                key="properties"
+                                disabled={personLoading}
+                            />
+                        </Tabs>
+                        {person && (
+                            <>
+                                {Object.keys(person.properties)
```

Comment:
This is duplicating frontend/src/lib/components/PropertyKeyInfo.js which we already duplicate badly in other places (see e.g. not being able to filter by props starting with $ in some modals).

We should consolidate this logic rather than pull this into more specific components.

Reply:
I'm actually using `PropertyKeyInfo.js` inside the component, the extra logic is for sorting purposes only. However, you're 100% right, I will add this logic to the main `PropertiesTable` component so it's reused across components.

Reply:
:heart: